src/tiled/commandmanager.cpp

```diff
@@ -81,28 +101,35 @@ void CommandManager::populateMenu(QMenu *menu, bool flag)
         if (!command.isEnabled)
             continue;
 
-        QAction *mAction = menu->addAction(command.name);
-        if (firstEnabledCommand && flag)
+        QAction *mAction = new QAction(this);
+
+        QString commandText = command.name;
```

Comment:
Please avoid the needless local variable. You can even pass this directly into the QAction constructor.